backend/zeno_backend/server.py

```diff
@@ -513,41 +505,41 @@ def get_projects(current_user=Depends(auth.claim())):
         user = select.user(current_user["username"])
         if user is None:
             return Response(status_code=status.HTTP_401_UNAUTHORIZED)
-        return select.projects(user)
+        return select.projects(user, ProjectsRequest())
```

Comment:
Why don't we make this a param of the function as well? What if I want limits here, too?